packages/web-forms/src/components/widgets/LikertWidget.vue

```diff
@@ -0,0 +1,86 @@
+<script setup lang="ts">
+import type { SelectNode } from '@getodk/xforms-engine';
+import RadioButton from '../widgets/RadioButton.vue';
+
+defineProps<{question: SelectNode}>();
+
+</script>
+
+<template>
+	<div class="likert">
+		<RadioButton :question="question" />
```

Comment:
I was a little surprised by the clickable area with the `likert` appearance. At first it seemed like the grey line was specifically made clickable, and then I realized that it is the full `<label>` containing each option... which even extends all the way to the left side of the leftmost option (and right/rightmost). It's really not obvious at all where the clickable area for one option ends, and the next begins. Worth considering:

- For better experience with pointer inputs (mouse), give some sort of indication of the target with a hover state
- For better experience with touch input, leave at least a small gap of inert area between each option

Reply:
Incorporated the suggested changes.